### PR TITLE
Feature - 1066 Service actions - Open edit modal

### PR DESCRIPTION
**Pull request by aldipower**
This opens the service edit modal if I click on the edit button.

editmodal

@Poltergeist Thanx!

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @kennyt

**Comment by orlandohohmeier** [comment]
I get an error when switching to the json mode due wrong trigger submit type.

Type Error

**Comment by aldipower** [comment]
@orlandohohmeier This PR is actually only meant to open the modal. But yes, an error should be fixed.

**Comment by orlandohohmeier** [comment]
Alright! _(Should have read the description)_

File: src/js/components/ServiceDetail.js [hunk 1]
```diff
@@ -22,8 +24,22 @@ class ServiceDetail extends mixin(InternalStorageMixin, TabsMixin) {
     };
 
     this.state = {
-      currentTab: Object.keys(this.tabs_tabs).shift()
+      currentTab: Object.keys(this.tabs_tabs).shift(),
```
Comment (kennyt) [inline src/js/components/ServiceDetail.js]: why not just `[0]`?

File: src/js/components/ServiceDetail.js [hunk 2]
```diff
@@ -22,8 +24,22 @@ class ServiceDetail extends mixin(InternalStorageMixin, TabsMixin) {
...
+      isServiceFormModalShown: false
     };
+
+    this.onActionsItemSelection = this.onActionsItemSelection.bind(this);
```
Comment (kennyt) [inline src/js/components/ServiceDetail.js]: since there's now two, let's just do the `METHODS_TO_BIND` pattern

**Comment by aldipower** [comment]
I've took every PR related comment into account.

**Comment by orlandohohmeier** [comment]
LGTM! 💯

**Comment by Poltergeist** [comment]
LGTM :shipit: 
```

          |\___..--"/
   __..--``        /
  \_______________/   Ship It
```

**Pull request merged**